app/main/controllers/template/rtmedia-filters.php

```diff
@@ -766,24 +766,26 @@ function rtmedia_search_fillter_where_query( $where, $table_name, $join ) {
  * @return string
  */
 function rtmedia_search_fillter_join_query( $join, $table_name ) {
-
-	global $wpdb;
-	$posts_table = $wpdb->posts;
-	$terms_table = $wpdb->terms;
-	$term_relationships_table = $wpdb->term_relationships;
-	$term_taxonomy_table = $wpdb->term_taxonomy;
-	$search = ( isset( $_REQUEST['search'] ) ) ? sanitize_text_field( wp_unslash( $_REQUEST['search'] ) ) : '';
-	$search_by = ( isset( $_REQUEST['search_by'] ) ) ? sanitize_text_field( wp_unslash( $_REQUEST['search_by'] ) ) : '';
-
-	if ( '' != $search ) {
-			$join .= "INNER JOIN $posts_table as post_table ON ( post_table.ID = $table_name.media_id AND post_table.post_type = 'attachment')";
-
-		$request_url = explode( '/', $_SERVER['REQUEST_URI'] );
-		if ( ! empty( $search_by ) && 'attribute' === $search_by && ! in_array( 'attribute', $request_url )  ) {
-			$join .= " 	INNER JOIN $posts_table ON ( $posts_table.ID = $table_name.media_id AND $posts_table.post_type = 'attachment' )
-	                    INNER JOIN $terms_table ON ( $terms_table.slug IN ('" . $search . "') )
-	                    INNER JOIN $term_taxonomy_table ON ( $term_taxonomy_table.term_id = $terms_table.term_id )
-	                    INNER JOIN $term_relationships_table ON ( $term_relationships_table.term_taxonomy_id = $term_taxonomy_table.term_taxonomy_id AND $term_relationships_table.object_id = $posts_table.ID ) ";
+	if ( function_exists( 'rtmedia_media_search_enabled' ) && rtmedia_media_search_enabled() ) {
+		global $wpdb;
+		$posts_table = $wpdb->posts;
+		$terms_table = $wpdb->terms;
+		$term_relationships_table = $wpdb->term_relationships;
+		$term_taxonomy_table = $wpdb->term_taxonomy;
+		$search = ( isset( $_REQUEST['search'] ) ) ? sanitize_text_field( wp_unslash( $_REQUEST['search'] ) ) : '';
+		$search_by = ( isset( $_REQUEST['search_by'] ) ) ? sanitize_text_field( wp_unslash( $_REQUEST['search_by'] ) ) : '';
+
+		if ( '' != $search ) {
+				$join .= "INNER JOIN $posts_table as post_table ON ( post_table.ID = $table_name.media_id AND post_table.post_type = 'attachment')";
+
+			$request_uri = filter_input( INPUT_SERVER, 'REQUEST_URI', FILTER_SANITIZE_URL );
+			$request_url = explode( '/', $request_uri );
+			if ( ! empty( $search_by ) && 'attribute' === $search_by && ! in_array( 'attribute', $request_url ) ) {
```

Comment:
`! in_array( 'attribute', $request_url )` to ! ` in_array( 'attribute', $request_url, true )`